W-20282179-rtf-2.11-remove-following-kt (#1602)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/configure-local-registry.adoc

+
If you set up a local registry, you can upgrade to a version later than 1.12.0 without reinstalling Runtime Fabric.
+

> To add local registry to an existing Runtime Fabric instance, ensure that the installation is Helm-based xref:migrate-helm.adoc[] and perform the steps on these section to add local registry.

Reviewer: ```suggestion
To add local registry to an existing Runtime Fabric instance, ensure that the installation is Helm-based xref:migrate-helm.adoc[] and perform the steps in this section to add local registry.
```

MuleSoft doesn’t provide additional software to synchronize third party dependencies and images. Push all the dependency images for your Mule app to your local registry before you start the deployments.

> * If you use a local registry that needs authentication, synchronization and propagation of a pull secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull secret across different namespaces if your secret uses these label:

Reviewer: ```suggestion
* If you use a local registry that needs authentication, synchronization and propagation of a pull secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull secret across different namespaces if your secret uses this label:
```

Ensure that your xref:access-management::permissions-by-product.adoc#runtime-manager[Access Management permission for Runtime Manager] is set to *Read Runtime Fabric*, which enables you to query Runtime Fabrics in your organization.

> . In your Unix shell, run these script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:

Reviewer: ```suggestion
. In your Unix shell, run this script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
```

--
To synchronize the dependencies images manually, follow these steps:

> . Log in to your private container registry and run these command, replacing `docker-server` where appropriate:

Reviewer: ```suggestion
. Log in to your private container registry and run this command, replacing `docker-server` where appropriate:
```

# rtfctl install ‘<activation_data>’ --image-pull-registry <docker-server>
----
+

> Alternatively, if you use authentication to access your registry, use these command:

Reviewer: ```suggestion
Alternatively, if you use authentication to access your registry, use this command:
```

. xref:install-helm.adoc#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager].
. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
. Obtain your private `docker-server`, `docker-username`, and `docker-password`.

> . Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running these command in your Unix shell:

Reviewer: ```suggestion
. Follow the *Helm* path instructions in Runtime Manager to create a secret with your docker server name, username, and password by running this command in your Unix shell:
```
